### DISCUSSION TITLE
when putting the XML code, the screens that are in the design disappear

### DISCUSSION THREAD

**Discussion by mora8292** [ImaginativeShohag/Why-Not-Image-Carousel#100] [Q&A]
`<?xml version="1.0" encoding="utf-8"?>
<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:carousel="http://schemas.android.com/apk/res-auto"
    xmlns:tools="http://schemas.android.com/tools"
    android:layout_width="match_parent"
    android:layout_height="match_parent"
    android:orientation="vertical"
    tools:context=".MainActivity2">

    <org.imaginativeworld.whynotimagecarousel.ImageCarousel
        android:id="@+id/carousel"
        android:layout_width="match_parent"
        android:layout_height="match_parent" />
### I already tried to see the error with chatgpt and it tells me that the build.gradle code is fine

**Comment by ImaginativeShohag** [comment]
Give a fixed height. If that also does not fix the issue, share the full layout if possible.

**Comment by mora8292** [comment]
![image](https://user-images.githubusercontent.com/104395775/226474581-05817334-504e-4c3d-8256-767975faf857.png)

The activities still do not appear for me

And this is what I have in the build.gradle
`plugins {
    id 'com.android.application'
}

android {
    namespace 'com.example.login'
    compileSdk 33


    defaultConfig {
        applicationId "com.example.login"
        minSdk 21
        targetSdk 33
        versionCode 1
        versionName "1.0"

        testInstrumentationRunner 'androidx.test.runner.AndroidJUnitRunner'
    }

    buildTypes {
        release {
            minifyEnabled false
            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
        }
    }
    compileOptions {
        sourceCompatibility JavaVersion.VERSION_1_8
        targetCompatibility JavaVersion.VERSION_1_8
    }
}

dependencies {
    implementation fileTree(dir: 'libs', include:['*.jar'])
    def nav_version = "2.5.3"

    implementation "androidx.navigation:navigation-fragment:$nav_version"
    implementation "androidx.navigation:navigation-ui:$nav_version"

    implementation 'com.google.android.material:material:1.8.0'


    implementation 'androidx.appcompat:appcompat:1.6.1'
    implementation 'androidx.constraintlayout:constraintlayout:2.1.4'
    testImplementation 'junit:junit:4.13.2'
    androidTestImplementation 'androidx.test.ext:junit:1.1.5'
    androidTestImplementation 'androidx.test.espresso:espresso-core:3.5.1'

    // Material Components for Android. Replace the version with the latest version of Material Components library.
    implementation 'com.google.android.material:material:1.8.0'

    // Circle Indicator (To fix the xml preview "Missing classes" error)
    implementation 'me.relex:circleindicator:2.1.6'

    implementation 'org.imaginativeworld.whynotimagecarousel:whynotimagecarousel:2.1.0'
    

}`

And this and this in the settings.gradle
`pluginManagement {
    repositories {
        google()
        mavenCentral()
        gradlePluginPortal()

    }
}
dependencyResolutionManagement {
    repositoriesMode.set(RepositoriesMode.FAIL_ON_PROJECT_REPOS)
    repositories {
        google()
        mavenCentral()
        maven {url'https://jitpack.io'}
    }

}
rootProject.name = "login"
include ':app'
`
I forgot to mention that I am new to Android Studio and just starting out
